File: Java/프로그래머스/0/181876. 소문자로 바꾸기/소문자로 바꾸기.java
class Solution {
    public String solution(String myString) {
        char ch[] = myString.toCharArray();
        for(int i=0; i<ch.length;i++){
            if(ch[i]>=65 && ch[i]<=90)
            ch[i]= Character.toLowerCase(ch[i]);
        }
        String answer= String.valueOf(ch);
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181876. 소문자로 바꾸기/소문자로 바꾸기.js
function solution(myString) {
    var answer = '';
    for(var i=0; i<myString.length;i++){
        if(myString.charAt(i)>'z' && myString.charAt(i)<'a'){
            answer+=myString.charAt(i).toLowerCase();
        }
        else {
            answer+=myString.charAt(i).toLowerCase();
        }
    }
    return answer;
}